lib/themeroller.js

```diff
@@ -112,17 +112,33 @@ function ThemeRoller( baseThemeCss, vars, options ) {
 
 	} else {
 
-		// For version >= 1.10.0, filter has its own separate line and variable name.
-		opacityFix = function( opacity ) {
-			return ( opacity / 100 ).toString().replace( /^0\./, "." );
-		};
-		opacityFilter = function( opacity ) {
-			return "Alpha(Opacity=" + opacity + ")";
-		};
-		vars.opacityFilterOverlay = opacityFilter( vars.opacityOverlay );
-		vars.opacityFilterShadow = opacityFilter( vars.opacityShadow );
-		vars.opacityOverlay = opacityFix( vars.opacityOverlay );
-		vars.opacityShadow = opacityFix( vars.opacityShadow );
+		if ( options.version && semver.lt( options.version, "1.13.0" ) ) {
+
+			// For version >= 1.10.0 < 1.13.0, filter has its own separate line and variable name.
+			opacityFix = function( opacity ) {
+				return ( opacity / 100 ).toString().replace( /^0\./, "." );
+			};
+			opacityFilter = function( opacity ) {
+				return "Alpha(Opacity=" + opacity + ")";
+			};
+			vars.opacityFilterOverlay = opacityFilter( vars.opacityOverlay );
+			vars.opacityFilterShadow = opacityFilter( vars.opacityShadow );
+			vars.opacityOverlay = opacityFix( vars.opacityOverlay );
+			vars.opacityShadow = opacityFix( vars.opacityShadow );
+		}
+		else {
+			// For version >= 1.13.0, filter has its own separate line and variable name.
+			opacityFix = function( opacity ) {
+				return ( opacity / 100 ).toString().replace( /^0\./, "." );
+			};
+			opacityFilter = function( opacity ) {
+				return "alpha(opacity=\"" + opacity + "\")";
```

Comment:
There should be no quotes around the opacity value; quotes should encompass the whole value. That’s the main difference between `filter` & `-ms-filter` and UI 1.13 uses the latter one.